include/zephyr/dt-bindings/regulator/pca9420_i2c.h

```diff
@@ -145,6 +148,7 @@
 	2050000 0x16		/* LDO2 output voltage 2.050V. */\
 	2075000 0x17		/* LDO2 output voltage 2.075V. */\
 	2100000 0x18		/* LDO2 output voltage 2.100V. */\
+	2100000 0x19		/* LDO2 output voltage 2.100V. */\
```

Comment:
Should we be setting `1A-1F` to `2.1V`

Reply:
We could- my rationale was that we should assume this driver is the only consumer of the regulator, so these values will never be written (although they are valid)